apps/web/app/features/stacking/utils/types-preset-pools.ts

```diff
@@ -0,0 +1,76 @@
+export const enum NetworkInstance {
+  mainnet = 'mainnet',
+  testnet = 'testnet',
+  devnet = 'devnet',
+}
```

Comment:
As we're working through this, let's aim to remove the enums

Reply:
will refactor here and in other places